tap_clickup/schemas/team.json

```diff
@@ -1,58 +1,123 @@
+                },
+                "username": {
+                    "type": ["null", "string"]
+                },
+                "email": {
+                    "type": ["null", "string"]
+                },
+                "color": {
+                    "type": ["null", "string"]
+                },
+                "profilePicture": {
+                    "type": ["null", "string"]
+                },
+                "initials": {
+                    "type": ["null", "string"]
+                },
+                "role": {
+                    "type": ["null", "integer"]
+                },
+                "custom_role": {
+                    "type": ["null", "integer"]
+                },
+                "last_active": {
+                    "type": ["null", "string"]
+                },
+                "date_joined": {
+                    "type": ["null", "string"]
+                },
+                "date_invited": {
+                    "type": ["null", "string"]
+                }
             }
         },
         "members": {
-            "type": "array",
-            "items": [
-                {
-                    "type": "object",
+            "type": ["null", "object"],
+            "properties": {
+                "user": {
+                    "type": ["null", "object"],
+                    "properties": {
+                        "id": {
+                            "type": ["null", "integer"]
+                        },
+                        "username": {
+                            "type": ["null", "string"]
+                        },
+                        "email": {
+                            "type": ["null", "string"]
+                        },
+                        "color": {
+                            "type": ["null", "string"]
+                        },
+                        "profilePicture": {
+                            "type": ["null", "string"]
+                        },
+                        "initials": {
+                            "type": ["null", "string"]
+                        },
+                        "role": {
+                            "type": ["null", "integer"]
+                        },
+                        "custom_role": {
+                            "type": ["null", "integer"]
+                        },
+                        "last_active": {
+                            "type": ["null", "string"]
+                        },
+                        "date_joined": {
+                            "type": ["null", "string"]
+                        },
+                        "date_invited": {
+                            "type": ["null", "string"]
+                        }
+                    }
+                },
+                "invited_by": {
+                    "type": ["null", "object"],
                     "properties": {
-                        "user": {
-                            "$ref": "#/definitions/user"
```

Comment:
This should be added back to the json file, shouldn't have been removed as we now have a ref resolver.